docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+.. _tutorial-moving-a-data-volume:
+
+==============================
+Tutorial: Moving a Data Volume
+==============================
+
+In this tutorial (30 minutes)
+-----------------------------
+
+You will use Flocker to migrate a Docker container with its data volume from one host to another.
+The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
+
+You will need
+-------------
+
+1. Somewhere to install the Flocker client.
+Make sure you have **one** of the following on your machine:
+
+- OS X with `Homebrew <http://brew.sh/>`_ installed.
+- Ubuntu 14.04.
+
+2. Two hosts for two instances of Flocker.
```

Comment:
My preference is that we have just the VM option and link to the full instructions for getting started on cloud / physical hosts. Setting those up is a time consuming process (we plan to make a tool to help you) and so it doesn't fall under the bracket of an ~30m guide.